examples/aliases/aliases.py

```diff
@@ -41,7 +41,7 @@ def get_command(self, ctx, cmd_name):
         # will create the config object is missing.
         cfg = ctx.ensure_object(Config)
 
-        # Step three: lookup an explicit command aliase in the config
+        # Step three: lookup an explicit command alias in the config
```

Comment:
Can you please change “lookup” to “look up” while you’re fixing this?

Reply:
Sure thing

Reply:
Thanks for spotting that - fixed